cran-comments.md

```diff
@@ -0,0 +1,12 @@
+## This is a new release for Azure Machine Learning SDK in R.
+
+## Test environments
+* local: windows-x86_64-devel
+* azure devops: Ubuntu:16.04 LTS
+* r-hub: windows-x86_64-devel, ubuntu-gcc-release
+* win-builder: windows-x86_64-devel
+
+## R CMD check results
+0 errors | 0 warnings | 1 note
+
+There is 1 note about a "Found the following hidden files and directories: .azure-pipelines and .github". The .azure-pipelines directory contains the pipelines to run against our code. The .github directory contains templates for reporting a bug or requesting a new feature.
```

Comment:
Shall we add files under .github and .azure-pipeline to .Rbuildignore?

Reply:
good point. will add it

Reply:
Ln12 can be removed.